tests/nnstreamer_query/runTest.sh

```diff
@@ -18,6 +18,15 @@ fi
 # This is compatible with SSAT (https://github.com/myungjoo/SSAT)
 testInit $1
 
+# Skip query test temporarily because of ppa build failure.
+# Related issue: https://github.com/nnstreamer/nnstreamer/issues/3657
+OS=$(uname)
```

Comment:
This will skip `tizen` also.
How about using `cat /etc/os-release`?

Reply:
Updated!
Thank you :)